cinderblock/commitstatus.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python
```

Comment:
You could even have the docstring double as additional usage information for argparse by doing something like:

    '''
    this module is great and you should totally use it
    '''

   # [...]

    argparser = argparse.ArgumentParser(description=__doc__)